canonical-web-and-design/ubuntu.com
# Update Ubuntu Core 20 downloads on /download/raspberry-pi-core and /download/raspberry-pi

## Done

- Updated the way the Raspberry pi core images are downloaded with the nicer thank-you page by updating `releases.yaml` and the download links
- Also made them an include so the `/download/raspberry-pi-core` page doesn't make people go back to the `/download/raspberry-pi` page to find the download

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/raspberry-pi-core and http://0.0.0.0:8001/download/raspberry-pi
- See that the download go to the thank-you page and that looks ok with the images and verify checksums
- See that the buttons are shared on both pages
- See that the text is updated on the /raspberry-pi page


## Issue / Card

Fixes #10710
